docs/contributing/pull-requests/branching.md

```diff
@@ -0,0 +1,246 @@
+---
+sidebar_custom_props:
+  access: bitwarden
+---
+
+# Branching
+
+## Naming Convention
+
+To keep branches organized we adhere to a naming convention for branches. The convention splits
+branch names into 2 components, each separated by a `/`.
+
+1. The developers' pod abbreviation (e.g. `sm`)
+2. The Jira issue tag (e.g. `SM-1234`)
+
+```bash
+# Some examples
+sm/SM-1234
+ps/PS-4321-optional-description
+```
+
+This naming convention allows us to easily identify the type of work being done on a branch and will
+assist in identifying and tracking down stale branches.
+
+## Branching for Development
+
+The branching strategy for development depends upon whether the work will be managed using what we
+have termed a "Long-Lived Feature Branch" or not.
+
+:::note
+
+We use the term "feature" in this documentation to refer to any change to the codebase, whether it
+be for a new set of functionality, a bug fix, or refactoring existing code. It is not meant to
+exclude types of development that are not "features" per se.
+
+:::
```

Comment:
Should we just remove `feature` from this document? I think _Long-Lived Branch_ works just as well as _Long-Lived Feature Branch_

Reply:
I've been in several discussions that indicate we might be moving toward adopting a more "by the books" gitflow branching strategy. That's why I kept the "feature" nomenclature.  Also, one of the things I've heard from new engineers coming in is that our slightly different branching strategy takes some time to understand, so I was trying to build in the comfort of knowing that these can be thought of as "feature branches" in the terminology of gitflow.

Reply:
That's fair. Feature branches is a well known term.